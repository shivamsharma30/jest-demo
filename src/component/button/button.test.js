import React from 'react';
import { shallow } from 'enzyme';
import Button from './component';
const labelText = 'add';

test('should test Button component', () => {
  const wrapper = shallow(<Button label={labelText} />);
  expect(wrapper).toMatchSnapshot();
});

test('should test button label', () => {
  const wrapper = shallow(<Button label={labelText} />);
  console.log(wrapper, wrapper.text());
  expect(wrapper.text()).toEqual('add');
  console.log(wrapper, wrapper.text());
});
