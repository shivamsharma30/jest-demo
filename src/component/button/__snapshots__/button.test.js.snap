// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Button component 1`] = `
<Fragment>
  <button
    id="btn"
  >
    add
  </button>
</Fragment>
`;
